### PR TITLE
[DOC-3215] Harness delegate 79503 release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
Harness delegate 795xx release notes

For reviewers, previews are available:

[What's new](https://6480a5c5632a00067c8ffa5b--harness-developer.netlify.app/release-notes/whats-new)
[NG Delegate release notes](https://6480a5c5632a00067c8ffa5b--harness-developer.netlify.app/release-notes/delegate)
[CG Delegate release notes](https://6480a5c5632a00067c8ffa5b--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/fg-delegate)

June 7 updates:

NG - removed CDS-69745, CDS-68772, CDS-68312
added CDS-58625

CG - added CDS-70386
updated CDS-68821 

Note: Synced CD ticket descriptions with @roshnisarangadharan 

NG Ticket notes as of June 6:
20 issues
SRM-14724                      - what’s new
PL-38247, ZDS-42721.  - documented in last release
PL-38245                         - fixed issues
OIP-406                            - fixed issues
CI-8105, 44679                - fixed issues
CDS-70559                       - fixed issues
CDS-70410, 45105,45110,45128  - fixed issues
CDS-69745                     - fixed issues
CDS-69605                    - fixed issues
CDS-69576                    -  fixed issues
CDS-69342, 44344       - fixed issues
CDS-69063, 44206      - fixed issues
CDS-68772                    - fixed issues
CDS-68760                    - fixed issues
CDS-68683                    - fixed issues
CDS-68527 - drafted in early access, should this be in CD release notes?
CDS-68312, 43710        - documented in CD May 23, 2023 version 79306
CDS-59011                     - what’s new
CDS-58625, 42496       - Should this be in CD release notes?
CDS-56451, 41436        - fixed issues

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — COMMENTED** [review]
PL-38247

**Comment by brian-f-harness** [comment]
> PL-38247

This ticket was documented in the last release.

File: release-notes/whats-new.md [hunk 1]
```diff
@@ -17,9 +17,18 @@ Review the notes below to learn about the new features that are Generally Availa
...
-## Latest - June 06, 2023 
+## June 2023
+
+### June 07, 2023 Harness version 795xx, Harness Delegate version 795xx
+
+#### Harness Delegate
+
+- Kubernetes blue/green and canary deployments now support HorizontalPodAutoscaler and PodDisruptionBudget resources. (CDS-59011)
+
+- Converted Harness CD from an explicit to an implicit change source for Service Reliability Management. (SMP-14724)
```
Comment (pratmit) [inline release-notes/whats-new.md:28]: There is no ticket as SMP-14724. This should be SRM-14724.

File: release-notes/delegate.md [hunk 2]
```diff
@@ -30,6 +31,111 @@ Harness NextGen release 79306 includes the following changes for the Harness Del
 
 This release introduces the following new features and enhancements:
 
+- Kubernetes blue/green and canary deployments now support HorizontalPodAutoscaler and PodDisruptionBudget resources. (CDS-59011)
+
+- Converted Harness CD from an explicit to an implicit change source for Service Reliability Management. (SMP-14724)
```
Comment (pratmit) [inline release-notes/delegate.md:36]: Should be SRM-14724

**Review by pratmit — APPROVED** [review]
Contingent approval on the comments will be fixed.

File: release-notes/whats-new.md (hunk 1 above)
Comment (brian-f-harness) [reply release-notes/whats-new.md:28]: Nice catch. Thank you, @pratmit. Corrected with a commit.

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 3]
```diff
@@ -3,13 +3,27 @@
...
 :::info note
-Harness deploys changes to Harness SaaS clusters on a progressive basis. This means the features and fixes that these release notes describe might not be immediately available in your cluster. 
+Harness deploys changes to Harness SaaS clusters on a progressive basis. This means the features and fixes that these release notes describe might not be immediately available in your cluster.
 
 To identify the cluster that hosts your account, open Harness FirstGen and go to **Account Settings**. Then click **Overview**. The cluster is listed in **Harness Cluster Hosting Account**.
 :::
 
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md). For Self-Managed Enterprise Edition release notes, see [Self-Managed Enterprise Edition (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md).
 
+## Latest release - June 7, 2023, Harness version 795xx, Harness Delegate version 795xx
```
Comment (douglas-j-bothwell) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:13]: I assume you're going to update the release number when it's finalized....

File: release-notes/delegate.md [hunk 4]
```diff
@@ -30,6 +31,111 @@ Harness NextGen release 79306 includes the following changes for the Harness Del
...
+  <TabItem value="Early access">
+```
+
+This release includes the following early access features and enhancements:
+
+- You can now scale down the last successful stage environment you created using the blue/green deployment strategy to help efficiently manage resources. You can configure this step in the same stage or a different stage, depending your requirements. (CDS-68527)
+
+   Scaling down removes the HorizontalPodAutoscaler and PodDisruptionBudget resources. The Deployments, StatefulSets, DaemonSets, and Deployment Configs resources are scaled down.
+
+   You must configure the same infra definition as the blue/green deployment. Harness identifies resources from the release history, which is mapped to release name. A different infra definition configuration may lead to scaling down important resources.
```
Comment (douglas-j-bothwell) [inline release-notes/delegate.md:50]:
```suggestion
   You must configure the same infrastructure definition as the blue/green deployment. Harness identifies resources from the release history, which is mapped to release name. A different infrastructure definition configuration may lead to scaling down important resources.
```

**Review by douglas-j-bothwell — DISMISSED** [review]

**Review by pratmit — DISMISSED** [review]
LGTM

**Review by roshnisarangadharan — DISMISSED** [review]

**Review by roshnisarangadharan — DISMISSED** [review]

**Pull request merged**